package.json

```diff
@@ -31,6 +31,7 @@
   },
   "devDependencies": {
     "@oat-sa/expr-eval": "^1.3.0",
+    "@oat-sa/tao-core-libs" : "^0.1.0",
```

Comment:
I have a warning from `npm`, because `tao-core-libs` wants to have a `jquery@1.9.1`, but because in this `package.json`, you allow to `npm` to install ^1.9.1, my `npm` installed `jquery@1.12.4`.
npm WARN @oat-sa/tao-core-libs@0.1.0 requires a peer of jquery@1.9.1 but none is installed. You must install peer dependencies yourself.